### PR TITLE
Release 3.1.0

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,8 +4,19 @@ All notable changes to this project will be documented in this file.
...
+## [v3.0.1](https://github.com/voxpupuli/puppet-staging/tree/v3.0.1) (2018-01-03)
+[Full Changelog](https://github.com/voxpupuli/puppet-staging/compare/v3.0.0...v3.0.1)
+
+**Closed issues:**
+
+- Add an opts parameter for tar [\#142](https://github.com/voxpupuli/puppet-staging/issues/142)
 
+**Merged pull requests:**
+
+- Add untar\_opts parameter for extract [\#151](https://github.com/voxpupuli/puppet-staging/pull/151) ([ardichoke](https://github.com/ardichoke))
```
Comment (alexjfisher) [inline CHANGELOG.md]: Does this mean it should be 3.1.0?

Comment (ekohl) [reply CHANGELOG.md]: You're right, I'll label that as enhancement.

**Comment by ekohl** [comment]
Updated

**Comment by alexjfisher** [comment]
Any idea why the tests are failing?

**Comment by ekohl** [comment]
It's updating bundler but that version is incompatible with ruby 2.1.9 I think. We removed that in the modulesync so I'll update that too.

**Comment by ekohl** [comment]
https://github.com/voxpupuli/puppet-staging/pull/153

**Comment by ekohl** [comment]
💚

**Pull request merged**